pkg/datapath/linux/ipsec/ipsec_linux.go

```diff
@@ -930,6 +933,52 @@ func DeleteXfrm() error {
 	return ee.Error()
 }
 
+// DeleteXfrmWithReqID remove any XFRM policy or state from tables which matches the reqID
+func DeleteXfrmWithReqID(reqID int) error {
```

Comment:
Nit: According to Go conventions, I believe this should be `DeleteXFRMWithReqID`.